README.md

```diff
@@ -93,7 +93,7 @@ tarocli assets send --addr tarotb1q[...]tywpre3a
 
 ### API
 
-Taro exposes a GRPC (port 10029) and a REST (port 8089) API. Connections are encrypted with TLS and authenticated using macaroons.
+Taro exposes a GRPC (port 10029) and a REST (port 8089) API. Connections are encrypted with TLS and authenticated using macaroons. [The API is documented here.](https://lightning.engineering/api-docs/api/taro/) Further guides [can be found here](https://docs.lightning.engineering/lightning-network-tools/taro).
```

Comment:
micro-nit: move the full stop outside the link for the first sentence to be consistent with the second one?

Reply:
good _point_, thank you!